docs/error-reporting/project-setup/submission-actions.md

```diff
@@ -19,15 +19,15 @@ Submission actions can be configured under Project Settings:
 The following actions are supported:
 
 ### Reject Submission
-This action will reject and never store the error. This is useful for customers who want to throw away crashes for specific noisy users, old versions, unsupported platforms, hacked games, etc.
+This action will reject and never store the error. This is useful if you want to throw away crashes for specific noisy users, old versions, unsupported platforms, hacked games, etc.
 
 ### Drop Object and Index Only
-This action will drop the minidump or other object from the error submission so it is not counted towards storage quota. This is useful if the user has no need to retain the object for storage and external analysis.
+This action will process (index) the minidump and any other error object (including file attachments) and then delete (drop) it so it is not counted towards storage quota. This is useful if you don't need to retain the object for storage and external analysis.
 
 <img src={useBaseUrl('img/error-reporting/project-settings/submission-actions-drop-object-and-index-only.png')} alt="" />
 
 ### Probability Sampling
-Both "Reject Submission" and "Drop Object and Index Only" offer functionality to apply a created rule with some probability.  In the example below, the rule would filter out 20% of error reports whos error message equals "Test Error".  
+Both "Reject Submission" and "Drop Object and Index Only" offer functionality to apply a created rule with some probability. In the example below, the rule would filter out 20% of error reports where the error message equals "Test Error".  
```

Comment:
Style guide wants us to avoid quotation marks. 
```suggestion
Both **Reject Submission** and **Drop Object and Index Only** offer functionality to apply a created rule with some probability. In the example below, the rule would filter out 20% of error reports where the error message equals **Test Error**.  
```